neuralmonkey/encoders/sentence_encoder.py

```diff
@@ -80,20 +80,25 @@ def __init__(self,
             .format(self.name))
 
         with tf.variable_scope(self.name):
-            self._create_input_placeholders()
-            self._create_embedding_matrix()
+            with tf.name_scope('intput_embeddings'):
```

Comment:
hlavně jestli to chápu správně tak embedding matrix, jakožto proměnná bude ve scope encoder, a input placeholdery jakožto noudy budou v input embeddings. operace nad embedding matrix pak bude už asi správně v input embeddings.

Co  input_projection?